readme-template.hbs

```diff
@@ -0,0 +1,28 @@
+# bedrock-package-manager
+
+## Usage
+A package should register itself using a Bedrock event handler.
+```js
+bedrock.events.on('bedrock.init', () => {
+  brPackageManager.register({
+    alias: 'mockPackage',
+    meta: {
+      optional: 'meta values',
+    },
+    packageName: 'bedrock-package-manager-mock-package',
+    type: 'bedrock-mock-plugin',
+  });
+});
+```
+
+Use a registered package in an API.
+```js
+exports.getReports = async ({query = {}, storageApi}) => {
+  const pkg = brPackageManager.get({alias: storageApi, type: 'bedrock-mock-storage'});
```

Comment:
It would help improve understanding if these two examples matched.  On first read I wondered what this type had to do with the type above.

Reply:
done, please let me know if your concerns have been addressed.